diff --git a/WSPR-TX1.14.ino b/WSPR-TX1.14.ino
index 34ebe9a..ec1d91f 100644
--- a/WSPR-TX1.14.ino
+++ b/WSPR-TX1.14.ino
@@ -254,6 +254,8 @@ struct S_FactoryData
 #define WSPR_FREQ630m             47570000ULL   //630m      475.700kHz
 #define WSPR_FREQ2190m            13750000ULL   //2190m     137.500kHz
 
+// 1536Hz or +36Hz in Centihertz
+#define WSPR_FIXED_OFFSET  (100ULL * 36)
 
 #define FactorySpace true
 #define UserSpace    false
@@ -562,6 +564,7 @@ void setup()
   Serial.print((":"));
   Serial.println(SoftwareRevision);
 
+  Serial.println(F("{MIN} Modified by JJ1BDX"));
 
   //Blink StatusLED to indicate Reboot
   LEDBlink(16);
@@ -1344,7 +1347,9 @@ void DoWSPR ()
     {
       CurrentBand = 0;
       NextFreq(); //Cycle to next enabled band to transmit on
-      freq = freq + (100ULL * random (-100, 100)); //modify TX frequency with a random value beween -100 and +100 Hz
+      // Use fixed offset
+      freq = freq + WSPR_FIXED_OFFSET;
+      // freq = freq + (100ULL * random (-100, 100)); //modify TX frequency with a random value beween -100 and +100 Hz
       si5351aOutputOff(SI_CLK0_CONTROL);
       SendAPIUpdate (UMesCurrentMode);
 
@@ -1416,7 +1421,9 @@ void DoWSPR ()
                 }
                 GPSWakeUp();
                 NextFreq();// get the frequency for the next HAM band that we will transmit on
-                freq = freq + (100ULL * random (-100, 100)); //modify the TX frequency with a random value beween -100 and +100 Hz to avoid possible lengthy colisions with other users on the band
+                // Use fixed offset
+                freq = freq + WSPR_FIXED_OFFSET;
+                // freq = freq + (100ULL * random (-100, 100)); //modify the TX frequency with a random value beween -100 and +100 Hz to avoid possible lengthy colisions with other users on the band
                 smartdelay(3000);
               }
             }
